src/createBaseForm.js

```diff
@@ -382,6 +382,7 @@ function createBaseForm(option = {}, mixins = []) {
           removedList.forEach(this.clearField);
         }
         this.renderFields = {};
+        this.elementRendered = {};
```

Comment:
Thanks. Let's add additional test case about Component `setState()` to update for this : )

Reply:
Exactly what should the test case test?

Reply:
One`getFieldDecorator` in render. And render component twice without warning message.

Reply:
So the warning should appear only once? But what if the duplicate `name` is rendered only in the second render?

Reply:
Noop. I mean like:
```jsx
render() {
  return (
    getFieldDecorator('test')(<Input />)
  );
}

wrapper.instance().forceUpdate();
```

No warning get.